core/dbt/events/helpers.py

```diff
@@ -0,0 +1,16 @@
+import os
+from typing import List
+from dbt.constants import SECRET_ENV_PREFIX
+
+
+def env_secrets() -> List[str]:
+    return [v for k, v in os.environ.items() if k.startswith(SECRET_ENV_PREFIX) and v.strip()]
+
+
+def scrub_secrets(msg: str, secrets: List[str]) -> str:
+    scrubbed = msg
+
+    for secret in secrets:
+        scrubbed = scrubbed.replace(secret, "*****")
+
+    return scrubbed
```

Comment:
I'm not sure I love what I did here.  These were moved out because of some circular dependency issues.  I'm not sure they even belong in the `/events` dir since they're used in various places.

Reply:
Yeah, but I can't think of a great place either. If a superior location emerges, we can move it.